### ISSUE TITLE
Loading games and programs from the Nintendo Switch's folder.

### ISSUE THREAD

**Issue by LexLight** [Ryujinx/Ryujinx#2022]
#### Overview:
A function that can load games from the own Nintendo folder from the Nintendo Switch after being extracted from the MicroSD Card and put it on Ryujinx's SDMC folder, in order to being able to play our games and other programs downloaded digitally from the eShop from the game list, and use our own updates and DLC on there without that much hassle.

#### Smaller Details: 
Of course, we would probably need our own prod.keys and title.keys dumped from our own Nintendo Switch in order to be able to use this function, and of course, like I said before, put the Nintendo folder on Ryujinx's SDMC one.

#### Nature of Request:  
Addition.

### Why would this feature be useful?
Because, like I said before, it would make much less of a hassle for the end user to use not only their own digital games, programs, updates and DLCs, so that they wouldn't need to dump them, just use their own keys files and put their own Nintendo folder from their own MicroSD card on the SDMC folder of Ryujinx, and that's it.

**Comment by AcK77** [comment]
I don't think this is doable since it will need some NCM service's savedata to know which games/programs/applications is contained in SD file (and some other infos). Maybe a solution would be to implement an accurate NCM service which support the NCM savedata aswell.
